sites/hn.svelte.dev/package.json

```diff
@@ -5,18 +5,15 @@
 	"scripts": {
 		"dev": "vite dev",
 		"build": "vite build",
-		"preview": "vite preview",
-		"lint": "prettier --ignore-path .gitignore --check --plugin-search-dir=. .",
```

Comment:
why remove `lint` and `format`? we should probably have `check` as well:
```
"check": "svelte-kit sync && svelte-check --tsconfig ./jsconfig.json"
```

Reply:
Force of habbit from my own projects. Will add back

Reply:
Added back, with check